tests/scripts/run-xunit-perf.py

```diff
@@ -0,0 +1,454 @@
+    if os.path.isdir(sandboxDir):
+        shutil.rmtree(sandboxDir)
+    os.mkdir(sandboxDir)
+
+def set_perf_run_log(sandboxOutputDir):
+    if not os.path.isdir(sandboxOutputDir):
+        os.mkdir(sandboxOutputDir)
+    return os.path.join(sandboxOutputDir, "perfrun.log")
+
+def setup_optimization_level(env, optLevel):
+    my_env = env
+    if optLevel == 'min_opts':
+        my_env['COMPlus_JITMinOpts'] = '1'
+    elif optLevel == 'tiered':
+        my_env['COMPLUS_EXPERIMENTAL_TieredCompilation'] = '1'
+
+    return my_env
+
+def build_perfharness(coreclrRepo, sandboxDir, extension, dotnetEnv):
+    # Confirm dotnet works
+    dotnet = os.path.join(coreclrRepo, 'Tools', 'dotnetcli', 'dotnet' + extension)
+    run_args = [dotnet,
+            '--info'
+            ]
+    log(" ".join(run_args))
+
+    proc = subprocess.Popen(run_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=dotnetEnv)
+    (out, err) = proc.communicate()
+
+    if not proc.returncode == 0:
+        print_error(out, err, 'Failed to get information about the CLI tool.')
+
+    # Restore PerfHarness
+    perfHarnessPath = os.path.join(coreclrRepo, 'tests', 'src', 'Common', 'PerfHarness', 'PerfHarness.csproj')
+    run_args = [dotnet,
+            'restore',
+            perfHarnessPath]
+    log(" ".join(run_args))
+
+    proc = subprocess.Popen(run_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=dotnetEnv)
+    (out, err) = proc.communicate()
+
+    if not proc.returncode == 0:
+        print_error(out, err, 'Failed to restore PerfHarness.csproj')
+
+    # Publish PerfHarness
+    run_args = [dotnet,
+            'publish',
+            perfHarnessPath,
+            '-c',
+            'Release',
+            '-o',
+            sandboxDir]
+    log(" ".join(run_args))
+
+    proc = subprocess.Popen(run_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=dotnetEnv)
+    (out, err) = proc.communicate()
+
+    if not proc.returncode == 0:
+        print_error(out, err, 'Failed to publish PerfHarness.csproj')
+
+def copytree(src, dst):
+    for item in os.listdir(src):
+        s = os.path.join(src, item)
+        d = os.path.join(dst, item)
+        if os.path.isdir(s):
+            shutil.copytree(s, d)
+        else:
+            shutil.copy2(s, d)
+
+def main(args):
+    coreclrPerf, arch, operatingSystem, configuration, jitName, optLevel, runType, outputDir, stabilityPrefix, isScenarioTest, benchviewPath, isPgoOptimized, benchviewGroup, hasWarmupRun, collectionFlags, isLibrary, uploadToBenchview, better, sliceNumber, sliceConfigFile = validate_args(args)
+
+    platform = sys.platform
+    python = 'py'
+    if platform == 'linux' or platform == 'linux2':
+        platform = 'Linux'
+        python = 'python3'
+    elif platform == 'darwin':
+        platform = 'OSX'
+        python = 'python3'
+    elif platform == 'win32':
+        platform = "Windows_NT"
+
+    executable = ['cmd.exe', '/c'] if platform == 'Windows_NT' else []
```

Comment:
>'Windows_NT' [](start = 50, length = 12)

If we use sys.platform, then this will be win32 and we would still need `cmd.exe /c`